src/handlers/gestures/GestureDetector.tsx

```diff
@@ -148,6 +150,9 @@ function attachHandlers({
   // use setImmediate to extract handlerTags, because all refs should be initialized
   // when it's ran
   setImmediate(() => {
+    if (!mountedRef.current) {
+      return;
+    }
```

Comment:
This one didn't happen to me (yet), but I added it for good measure - we should probably never assume in an `setImmediate` call that the component did not unmount before.